### ISSUE TITLE
AttrNode has no `propertyDidChange()` method.

### ISSUE THREAD

**Issue by gigafied** [emberjs/ember.js#10502]
Using `ember-list-view` if I update a controller for a `parentView` and that `parentView` has an `AttrNode` child view, `Ember.View` code throws an error:

```
      _controllerDidChange: observer('controller', function() {
        if (this.isDestroying) { return; }

        this.rerender();

        this.forEachChildView(function(view) {
          view.propertyDidChange('controller');
        });
      }),
```

It's trying to call `propertyDidChange()` on `AttrNodes`. I've changed it to `view.propertyDidChange && view.propertyDidChange('controller');` as a workaround for now, but not sure what the best way of actually fixing it is.

I'm using Ember 1.10.0

**Comment by mixonic** [comment]
@gigafied I cane push a patch for this on 1.10.1

**Comment by mixonic** [comment]
@gigafied what does you template look like? I'm having trouble understanding how this would happen in the wild.

**Comment by gigafied** [comment]
I think it specifically has to do with `ember-list-view`. I know there is not an official release of list-view that supports Ember 1.10.0 so maybe it's an issue there.

My template is pretty simple:

```
    {{#if thumbnailUrl}}
        <img {{bind-attr src="thumbnailUrl"}} />
    {{/if}}
```

**Comment by mixonic** [comment]
@gigafied I'm trying to recreate this error in a test, but I'm worried that I won't actually fix your issue unless I have a reproduction from you. Can you try for a JSBin today? http://emberjs.jsbin.com/

**Comment by mixonic** [comment]
I am having a tough time finding a case where `_controllerDidChange` fires. It makes testing the fix pretty tough :-/

**Comment by gigafied** [comment]
Yeah, `ember-list-view` recycles views on scroll, and applies new context/controller to them. Trying to repro in a JSBin now.

**Comment by gigafied** [comment]
Ok, was able to repro : http://jsbin.com/qowawiwofu/1/edit?html,js,output

@stefanpenner should this be fixed in list-view or Ember?

**Comment by stefanpenner** [comment]
@gigafied can you also open a list-view issue? I need to find time to dig in to see whats borked

**Comment by billybonks** [comment]
so i'm having the same issue, but not when using the list view. i'm binding a url to a href attribute in a inline template. ill try build a fiddle that reproduces.

an overview would be 

```
controller ->
   myView: view.extend({
     template:Ember.HTMLBars.compile('<a {{bind-attr href=somevalue}}'></a>
   })
```

and in the controller template

`{{view myView}}`

**Comment by rwjblue** [comment]
Fixed by #10657

**Comment by gigafied** [comment]
I'm still seeing this in `1.11.3`, so I don't think it was fixed. I did post a JSBin of the issue, but for some reason the JSBin only works on `1.10.x` versions of Ember, due to another issue.

Unfortunately I can't upgrade to `1.13` as too much has been deprecated.
